sycl/test-e2e/Matrix/SPVCooperativeMatrix/element_wise_all_ops.cpp

```diff
@@ -0,0 +1,17 @@
+//==------------ element_wise_all_ops.cpp  - DPC++ joint_matrix-------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, aspect-ext_intel_matrix
+
+// RUN: %{build} -D__SPIRV_USE_COOPERATIVE_MATRIX -o %t.out
+// RUN: %{run} %t.out
+
+// TODO: Reenable, see https://github.com/intel/llvm/issues/14598
+// UNSUPPORTED: windows, linux
```

Comment:
this was enabled back, so here and below for all such disabled tests, they should be enabled back.